acceptance/config/nodes/ubuntu-lucid-i386.yaml

```diff
@@ -0,0 +1,18 @@
+HOSTS:
+  master:
+    roles:
+      - master
+    platform: el-7-x86_64
+    hypervisor: vcloud
+    template: redhat-7-x86_64
+  agent:
+    roles:
+      - agent
+    platform: ubuntu-lucid-i386
+    hypervisor: vcloud
+    template: ubuntu-1004-i386
```

Comment:
Unfortunately we're not consistent in our naming. The VM template is `ubuntu-1004-i386`. The BUILD_TARGETS as specified in jenkins AIO pipeline and puppet-agent is [ubuntu-10.04-i386](https://github.com/puppetlabs/puppet-agent/blob/master/configs/platforms/ubuntu-10.04-i386.rb). This yaml file and the `platform` are `ubuntu-lucid-i386`. It also looks like we sometimes use `amd64` vs `x86_64`... And the TEST_TARGET in AIO are something else...